frontend/packages/kubevirt-plugin/src/k8s/wrapper/vm/data-volume-wrapper.ts

```diff
@@ -86,19 +86,19 @@ export class DataVolumeWrapper extends K8sResourceObjectWithTypePropertyWrapper<
     return accessModes ? accessModes.map((mode) => AccessMode.fromString(mode)) : accessModes;
   };
 
-  setSize = (value: string | number, unit = 'Gi') => {
+  setSizeLegacy = (value: string | number, unit = 'Gi') => {
```

Comment:
```suggestion
  setPVCSize = (value: string | number, unit = 'Gi') => {
```
it's just PVC it's not getting deprecated or somethng